cluster-provision/images/vm-image-builder/customize-image.sh

```diff
@@ -8,7 +8,7 @@ function cleanup() {
 
   rm -rf "${CLOUD_INIT_ISO}"
   virsh destroy "${DOMAIN_NAME}" || true
-  virsh undefine "${DOMAIN_NAME}" || true
+  virsh undefine "${DOMAIN_NAME}" --nvram --remove-all-storage || true
```

Comment:
Why is it necessary to pass `--nvram`? isnt it enough to use `virsh undefine` as is?
Specifying `--remove-all-storage` will remove the provisioned image as well, isnt it?
Note that it is already being deleted at lines 5-7

Reply:
`--nvram` is necessary on aarch64, otherwise it fails, like there: https://github.com/vagrant-libvirt/vagrant-libvirt/issues/1371
`--remove-all-storage` can't hurt and will ensure all volumes are gone. I can remove it if it causes any issue.